operator/pkg/reconciliation/reconcile_racks.go

```diff
@@ -2081,6 +2095,10 @@ func (rc *ReconciliationContext) ReconcileAllRacks() (reconcile.Result, error) {
 		return recResult.Output()
 	}
 
+	if recResult := rc.CheckDecommissioningNodes(endpointData); recResult.Completed() {
```

Comment:
how did you pick this spot in the order for this part?

Reply:
I needed it to happen after the rack labels got set up